### PR TITLE
feat: add micro block time to oracle nested resources

### PR DESCRIPTION
**Pull request by sborrazas**
refs #1418

### CODE REVIEW

File: lib/ae_mdw/node/db.ex [hunk 1]
```diff
@@ -258,6 +258,9 @@ defmodule AeMdw.Node.Db do
     end
   end
 
+  @spec get_block(Blocks.block_hash()) :: key_block() | micro_block()
+  def get_block(block_hash), do: :aec_db.get_block(block_hash)
```
Comment (jyeshe) [inline lib/ae_mdw/node/db.ex:262]: :pray: for a motivation to be praised

Comment (jyeshe) [reply lib/ae_mdw/node/db.ex:262]: What you think about making public a function that calls `block_hash |> get_block() |> :aec_blocks.time_in_msecs()`?

Comment (jyeshe) [reply lib/ae_mdw/node/db.ex:262]: Besides encapsulating the Node it could simplify the tests as well.

**Review by jyeshe — COMMENTED** [review]
LGTM, minor suggest

File: lib/ae_mdw/node/db.ex (hunk 1 above)
Comment (sborrazas) [reply lib/ae_mdw/node/db.ex]: Good call. Done

**Review by jyeshe — APPROVED** [review]

**Pull request merged**